Bottom: 7ad3499de95de793768f2232b79f7e5f84c4da3d
Top:    2597e576f584e2171b6b31f34421d2ce94fb49cc
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-12-19 17:26:45 +0800

Refresh of xdp-introduce-xdp-free-func

---

diff --git a/include/linux/filter.h b/include/linux/filter.h
index 3ac76dd..74e504a 100644
--- a/include/linux/filter.h
+++ b/include/linux/filter.h
@@ -441,7 +441,7 @@ struct xdp_buff {
 	void *data_hard_start;
 	void *ring;
 	void *private;
-	struct net_device *netdev;
+	struct net_device *dev;
 	int (*free) (const void *ring, const void *private);
 };
